pkg/frontend/querymiddleware/model_extra.go

```diff
@@ -92,14 +91,13 @@ func (q *PrometheusRangeQueryRequest) WithEstimatedSeriesCountHint(count uint64)
 	return &newRequest
 }
 
-// LogToSpan logs the current `PrometheusRangeQueryRequest` parameters to the specified span.
+// LogToSpan writes the current `PrometheusRangeQueryRequest` parameters to the specified span tags
+// ("attributes" in OpenTelemetry parlance).
 func (q *PrometheusRangeQueryRequest) LogToSpan(sp opentracing.Span) {
```

Comment:
should we rename this method now? It doesn't log to the span anymore (same with PrometheusInstantQueryRequest.LogToSpan)

Reply:
That's a good idea, thanks! Someone also privately noted to me that I should look into tag value length limits.